### DISCUSSION TITLE
How to find what versions of Docker and Python are used in the CI environment?

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#113] [Q&A] [answered]
@stebo85 @aswinnarayanan How can users find what versions of Docker and Python are used in the CI environment?

It is mentioned here https://neurodesk.github.io/developers/new_tools/add_tool/#build-container, but there is no link or additional information. I can add more details, but don't know myself.

**Comment by stebo85** [comment, marked as answer]
Dear Oren,

it's not necessary to exactly match the versions and I now clarified it in the documentation.

Kind regards
Steffen

**Comment by civier** [comment]
@stebo85 Maybe it is not absolutely necessary, but recommended for sure (especially if somebody sets up a new environment). I suggest that we recommend (without making it mandatory) to have exactly the same versions, and explain how to find the version numbers.

Another issue is the Python version required for Neurodocker. The current Neurodocker needs Python 3.7 and above, so that should be a mandatory requirement after all. I sent a pull request on that regard.

**Reply by stebo85** [reply]
ok, then clarify what version worked for you and add this to the documentation.
